src/views/pages/ManagePeople/index.vue

```diff
@@ -36,18 +36,20 @@
           <div class="md-gutter">
             <div class="md-layout-item md-small-size-100">
               <h3 class="md-subheading"><b>Category</b></h3>
-              <md-radio v-model="form.category" value="farmer">Farmer</md-radio>
-              <md-radio v-model="form.category" value="trader">Trader</md-radio>
-              <md-radio v-model="form.category" value="admin">Admin</md-radio>
-              <md-radio v-model="form.category" value="monitor">Monitor</md-radio>                 
+              <md-radio v-model="form.category" value="farmer" name="person-type" v-validate="'required'">Farmer</md-radio>
```

Comment:
Why is just this required?

Reply:
Only one `v-validate` is required for radio buttons given they have the same name.